### PR TITLE
Badr

### PR DESCRIPTION
**Pull request by TERY-TERY-TERY**
I added some new variables

### CODE REVIEW

File: src/BodyProgramming/Badr.java [hunk 1]
```diff
@@ -31,5 +31,7 @@ public static void main(String[] args) {
         System.out.println(p);
         double q = 1.56642538343;
         System.out.println(q);
+        int a = 7;
+        int b = 8;
```
Comment (Mohamed-Khaled-El-Sayed) [inline src/BodyProgramming/Badr.java]: why did you add 2 variables not only one ?

**Review by Mohamed-Khaled-El-Sayed — CHANGES_REQUESTED** [review]

File: src/BodyProgramming/Badr.java (hunk 1 above)
Comment (TERY-TERY-TERY) [reply src/BodyProgramming/Badr.java]: Done, I removed variable

**Review by Mohamed-Khaled-El-Sayed — APPROVED** [review]
